Adding  docs for the `k6/experimental/websockets` module (#933)
grafana/k6-docs: src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets/10 WebSocket/20-WebSocket-ping--.md

---
title: 'WebSocket.ping()'
excerpt: 'Send a ping. Ping messages can be used to verify that the remote endpoint is responsive.'
---

Send a ping. Ping messages can be used to verify that the remote endpoint is responsive.

### Example

> _A k6 script that will initiate WebSocket connection sends a ping and closes it using `onopen` handler. We also should see `connection is alive` since `pong` event should be automatically emitted._

Reviewer: ```suggestion
_A k6 script that initiates a WebSocket connection, sends a ping, and closes it using the `onopen` handler. The console should log `connection is alive`, since the recipient should automatically emit the `pong` event._
```
